src/app/components/error/index.html

```diff
@@ -1,13 +1,232 @@
+    <sky-demo-page-content sectionHeading="Usage">
+        
+      <h3 class="sky-text-success">Use when</h3>
+        
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Use error messages to communicate system statuses to users when the system encounters unexpected errors.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-1.png" />
+                <div class="caption caption-success sky-text-success"> Do use error messages when system errors occur.</div>
+            </div>
+          </sky-docs-visual-example-image>       
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Use error messages when the content inside modals fails to load.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-8.png" />
+                <div class="caption caption-success sky-text-success"> Do use error messages when system errors occur when opening modals.</div>
+            </div>
+          </sky-docs-visual-example-image>       
+        </sky-docs-visual-example>
+        
+        <h3 class="sky-text-danger">Don't use when</h3>
+        
+        <sky-docs-visual-example>
+            
+            <sky-docs-visual-example-text>
+              <p>Don’t use error messages to display information about background processes. Use <a routerLink="/components/toast">toasts</a> instead.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div class="sky-margin-stacked-content-section">
+                  <img src="~/assets/img/guidelines/error/error-usage-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use error messages for background processes.</div>
+                </div>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-7.png" />
+                    <div class="caption caption-success sky-text-success">Do use <a routerLink="/components/toast">toasts</a> to inform users about background processes.</div>
+                </div>
+            </sky-docs-visual-example-image>
 
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Don’t use error messages when the status of specific content requires user attention, but there is no system error. To notify users of a broken status on a workflow or record, use <a routerLink="/components/alert">alerts</a> instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-content-section">
+              <img src="~/assets/img/guidelines/error/error-usage-3.png" />
```

Comment:
it took me a minute to figure out what was going on in this example. it may be helpful to show the full page context.

Reply:
Is this still an issue? It looks to me like the images now show a full-page context, but just making sure.